### ISSUE TITLE
Can we add customized message to FindLiteral?

### ISSUE THREAD

**Issue by showpune** [openrewrite/rewrite#2654]
To define the FindLiteral as below

```
  - com.azure.spring.migration.openrewrite.java.search.FindLiterals:
      pattern: "[A-z]:([\\\\][^\\n\\t]+)+|(\\\\\\\\([^\\\\\\,\\n\\t]+)\\\\\\S+)+"
      mark: No window file url
```

Input Code:

`File file = new File("c:\\temp\\d.text");`

Output Code
File file = new File(/*~~(No window file url)~~>*/"c:\\temp\\d.text");

**Comment by timtebeek** [comment]
hi @showpune ; I don't recognize the `com.azure` prefix to your recipe as one of ours.

We do have a likely similar [OpenRewrite recipe](https://docs.openrewrite.org/reference/recipes/java/search/findliterals) at [org.openrewrite.java.search.FindLiterals](https://github.com/openrewrite/rewrite/blob/main/rewrite-java/src/main/java/org/openrewrite/java/search/FindLiterals.java#L75). Right now that returns a `SearchResult` without a decription, but I like your suggestion of adding an optional description. Is that something you would want to contribute, [possibly based on your work here](https://github.com/Azure/azure-spring-rewrite/blob/main/src/main/java/com/azure/spring/migration/openrewrite/java/search/FindLiterals.java)?

**Comment by showpune** [comment]
I am not authoized to create branch for https://github.com/openrewrite/rewrite

**Comment by timtebeek** [comment]
That's correct; we ask our contributors to create a fork of this project through the buttons at the top right corner.
![image](https://user-images.githubusercontent.com/1027334/212848538-546be48a-4147-413c-bc86-d8f1c2e8ec8f.png)

You can then clone that fork locally; make any changes that you want on a new branch and push that branch to your fork.
You should then get a prompt from GitHub whether you want to create a pull request on this repository with your changes.

Then once you've opened a pull request we can review your changes and hopefully adopt them into this code repository.

More information on that flow can be found at:
https://docs.github.com/en/get-started/quickstart/fork-a-repo
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks
Hope this helps!

**Comment by timtebeek** [comment]
Or did you mean to say your employer does not allow you to contribute here? Its's a bit unclear to me what you meant...
